spec/jobs/import_data_from_rapids_job_spec.rb

```diff
@@ -98,6 +98,35 @@
       end
     end
 
+    context "when receiving an existing standard" do
+      it "does not create a duplicated record" do
+        stub_get_token!
+        create(:registration_agency, for_state_abbreviation: "MI", agency_type: :oa)
+
+        occupation_standard_response = create_list(
```

Comment:
```suggestion
        occupation_standard_response = create_pair(
```
and can remove the "2" on line 108.